changelog/+diff-update.changed.md

```diff
@@ -0,0 +1 @@
+Update DiffUpdate mutation to return the id of the task when `wait_until_completion` is False.
```

Comment:
We could mention that `wait_for_completion` under the data object is deprecated and has been replaced with the new option. Should also create an issue to switch this in the frontend.